src/main/java/com/example/projectfirst/connector/exception/APIPConnectorAlreadyExistsException.java

```diff
@@ -0,0 +1,7 @@
+package com.example.projectfirst.connector.exception;
+
+public class APIPConnectorAlreadyExistsException extends RuntimeException{
```

Comment:
Add intermediate exception in inheritance chain 
APIPConnectorAlreadyExistsException extends RuntimeException implements APIPGeneralException